extensionsGallery-insider.json

```diff
@@ -2961,6 +2961,62 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "61",
+					"extensionName": "Visualization",
```

Comment:
Thanks for building this extension!  There is a little room for confusion in the naming, as we have an extension category for visualization, which currently use to describe data visualization.  See the product notification below for context.  Since this extension is a schema diagramming tool it likely wouldn't meet the definition of what we currently call a visualizer in the product.  Perhaps naming something like `Schema Visualizer` or something else that isn't ambiguous with existing product terminology would make more sense.

![image](https://user-images.githubusercontent.com/599935/74072098-97569500-49ba-11ea-8c66-d46a9a1d5075.png)

Reply:
I did not knew that was an category. I´ve changed the name of the extention.